src/main/java/nextstep/optional/ComputerStore.java

```diff
@@ -21,6 +23,10 @@ public static String getVersion(Computer computer) {
     }
 
     public static String getVersionOptional(Computer computer) {
-        return null;
+        Optional<Computer> computerOptional = Optional.ofNullable(computer);
+        return computerOptional.map(Computer::getSoundcard)
```

Comment:
computerOptional 변수 없이 바로 리턴해도 좋을 것 같아요
```suggestion
        return Optional.ofNullable(computer)
                .map(Computer::getSoundcard)
```